aiven/client/cli.py

```diff
@@ -320,8 +322,18 @@ def collect_user_config_options(self, obj_def, prefix=""):
         opts = {}
         for prop, spec in sorted(obj_def.get("properties", {}).items()):
             full_name = prop if not prefix else (prefix + "." + prop)
-            if spec["type"] == "object":
+            types = spec["type"]
+            if not isinstance(types, list):
+                types = [types]
+            # "object" or ["object", "null"]
+            if "object" in types:
                 opts.update(self.collect_user_config_options(spec, prefix=full_name))
+                if "null" in types:
+                    # allow removing option with empty value
```

Comment:
Should we rather have some explicit switch for specifying that a value is to be deleted? It is possible that both null and empty string are valid values and making empty string mean that the value should be set to null can cause problems later on.

Reply:
Yes I guess "service update -d <configuration> <service>" could work better.